### ISSUE TITLE
Revisar y corregir

### ISSUE THREAD

**Issue by saantifranco** [Batta32/VanTrackDatabase#2]
- sp_get_recorridos: revisar join de recorridos con paradas, porque si un recorrido no tiene paradas hoy no se está mostrando como resultado del sp
- Revisar definicion tabla dia campo descripcion en todos lados, no tenia "dia_"
- Qué ganamos haciendo que la empresa sea un usuario? Realmente va a tener todos los datos del usuario y en algún momento los vamos a mostrar para algo?

**Comment by saantifranco** [comment]
Por qué estamos "Reservando" un servicio + hora + dia y no un registro de "programacion_servicio"?

**Comment by saantifranco** [comment]
Por qué la empresa no tiene calificacion? Me parece todavia más relevante que la del conductor

**Comment by Batta32** [comment]
> Qué ganamos haciendo que la empresa sea un usuario? Realmente va a tener todos los datos del usuario y en algún momento los vamos a mostrar para algo?

No, no se va a mostrar en ningun momento, pero es el usuario a la empresa que vamos a proveer en base a los datos de alta que nos den ellos.

> Por qué la empresa no tiene calificacion? Me parece todavia más relevante que la del conductor

Dale, agrego la misma estructura que tiene el conductor para la empresa

El resto lo verifico y hago las correcciones necesarias.
@saantifranco
